### PR TITLE
Update Login response codes for gated content

### PR DESCRIPTION
**Pull request by bguyza**
This PR updates response codes to include `213` for any login API.  Specifically:
- /api/login: `login-response-codes.adoc`
- /api/two-factor/login: `login-response-codes.adoc`
- /api/identity-provider/login: `_identity-provider-login-response-body.adoc`
- /api/passwordless/login: `login-response-codes.adoc`

### CODE REVIEW

File: site/docs/v1/tech/apis/registrations.adoc [hunk 1]
```diff
@@ -213,6 +213,11 @@ link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[ty
 [method]#POST# [uri]#/api/user/verify-registration/`\{verificationId\}`#
 --
 
+[WARNING.deprecated]
+====
+Using [field]#verificaitonId# in the URL segment is deprecated as of version 1.27.0 and may be removed in the future. This API now takes this field in the request body.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/registrations.adoc:218]:
```suggestion
Using [field]#verificaitionId# in the URL segment is deprecated as of version 1.27.0 and may be removed in the future. This API now takes this field in the request body.
```

File: site/docs/v1/tech/apis/users.adoc [hunk 2]
```diff
@@ -790,6 +790,11 @@ link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[ty
 [method]#POST# [uri]#/api/user/verify-email/`\{verificationId\}`#
 --
 
+[WARNING.deprecated]
+====
+Using [field]#verificaitonId# in the URL segment is deprecated as of version 1.27.0 and may be removed in the future. This API now takes this field in the request body.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/users.adoc:795]:
```suggestion
Using [field]#verificationId# in the URL segment is deprecated as of version 1.27.0 and may be removed in the future. This API now takes this field in the request body.
```

Comment (mooreds) [reply site/docs/v1/tech/apis/users.adoc:795]: You need to add a Request Body section now, correct?

File: site/docs/v1/tech/apis/registrations.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/registrations.adoc:218]: You need to add a Request Body section now, correct?

File: site/docs/v1/tech/apis/_login-response-codes.adoc [hunk 3]
```diff
@@ -27,7 +27,10 @@ include::../../../src/json/login/login-change-password-response.json[]
 ----
 
 |212
-|The user's email address has not yet been verified. The response will contain the User object that was authenticated.
+|The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#emailVerificationId# that was generated for the user.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/_login-response-codes.adoc:30]: Is it worth adding a sample response, like the 242 and 203 responses have?

File: site/docs/v1/tech/apis/_login-response-codes.adoc [hunk 4]
```diff
@@ -27,7 +27,10 @@ include::../../../src/json/login/login-change-password-response.json[]
...
+
+|213
+|The user's registration has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#registrationVerificationId# that was generated for the user.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/_login-response-codes.adoc:33]: Is it worth adding a sample response, like the 242 and 203 responses have?

File: site/docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc [hunk 5]
```diff
@@ -25,7 +25,10 @@ include::../../../../src/json/login/login-change-password-response.json[]
 |The login is pending action by the end user. Wait and try the request again.
 
 |212
-|The user's email address has not yet been verified. The response will contain the User object that was authenticated.
+|The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#emailVerificationId# that was generated for the user.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc:28]: Is it worth adding a sample response, like the 242 and 203 responses have?

File: site/docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc [hunk 6]
```diff
@@ -25,7 +25,10 @@ include::../../../../src/json/login/login-change-password-response.json[]
...
+
+|213
```
Comment (mooreds) [inline site/docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc:30]: Is it worth adding a sample response, like the 242 and 203 responses have?

File: site/docs/v1/tech/apis/login.adoc [hunk 7]
```diff
@@ -279,7 +279,10 @@ include::docs/src/json/login/login-change-password-response.json[]
 ----
 
 |212
-|The user's email address has not yet been verified. The response will contain the User object that was authenticated.
+|The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#emailVerificationId# that was generated for the user.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/login.adoc:282]: Is it worth adding a sample response, like the 242 and 203 responses have?

File: site/docs/v1/tech/apis/login.adoc [hunk 8]
```diff
@@ -279,7 +279,10 @@ include::docs/src/json/login/login-change-password-response.json[]
...
+
+|213
+|The user's registration has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#registrationVerificationId# that was generated for the user.
```
Comment (mooreds) [inline site/docs/v1/tech/apis/login.adoc:285]: Is it worth adding a sample response, like the 242 and 203 responses have?

**Review by mooreds — CHANGES_REQUESTED** [review]

**Comment by bguyza** [comment]
@mooreds
- Updated fields to include a request body. It actually includes a oneTimeCode field too so I'm glad you called that out.
- I was going back and forth with regards to adding sample responses for 212 and 213.  The reason I decided to omit them was because the response contains the entire User object with the emailVerificationId or registrationVerificationId added to the json and it takes up a lot of real-estate on the page.  Since the existing 212 didn't have an example with the User object I thought the addition of one more field didn't necessarily warrant adding large example responses.

If you think its best to include it, I can go ahead and add.  Thoughts?

**Comment by mooreds** [comment]
@bguyza makes sense to not include the body in that case.

**Pull request merged**